### ISSUE TITLE
v2.1.1 Chrome extension - Not save video 

### ISSUE THREAD

**Issue by AutopilotNN** [puemos/hls-downloader-web-extension#93]
Windows 7 x64
Clean install Chome Portable 81.0.4044.129
Install HLS Downloader 2.1.1

Open: 
https://rutube.ru/list/premier/video/568f68477d1607a8fc71b2ee7b32892c/
Im have free account

Video is viewed and HLS Downloader open playlist, but save only small video ~460MB. 

If select a maximum size of 1920x1080, the video pieces are saved to the "blob_storage/2ce287c2-ff12-4875-8376-9d318bc352b6" folder, but you do not save them to a single file after downloading.
The large video had a file size 1,5GB. On the Download tab, when you move the cursor over the Save button, a crossed-out circle icon appears after 100%. It is impossible to keep.
And also - the extension saves all dimensions from the playlist to one folder.

The previous version of "HLS Download" worked perfectly.

**Comment by puemos** [comment]
Alright, I will take a look, thanks!

**Comment by merlinoonteweb** [comment]
Sometimes doesn't start the file manager windows at the end.
Just 100% downloaded.
What is the temp folder where the file is downloaded?
May I saved manually?

Sometimes when the file is saved and length and size are ok, VLC recognizes only sound but I've tried also Windows Media Player and it opens the file perfectly.
So something in the reconstruction of the file seems to be different.

**Comment by artik00** [comment]
I got this too

**Comment by AutopilotNN** [comment]
> What is the temp folder where the file is downloaded?
> May I saved manually?

profile\Default\blob_storage
Pieces of video may have numbers not in order in the video, but in order of download:
set in setting of HLS Downloader "Concurrency = 1"

**Comment by merlinoonteweb** [comment]
Thanks.
I see.
My destination is AppData\Local\Google\Chrome\User Data\Default\blob_storage
The program built a folder. Inside 257 fragment.
Rebuild without a problem.
When I ask to download a second file no new folder was built.
New fragments start n. 258 to 824 but seem the PC was unable to reassemble them.
The clear command does not clean the fragments
It is correct?

**Comment by merlinoonteweb** [comment]
Here it is the source of videos.
The full HD needs from 566 to 441 fragments to match 100%. I don't understand why.
The 720 HD file needs 257 fragments and was downloaded well.
https://www.paramountnetwork.it/episodi/ri7jr5/il-giovane-ispettore-morse-cantico-stagione-4-e-2

**Comment by aricjohnson** [comment]
This has been happening for me also. Usually not the first video, but the second or third.

**Comment by aricjohnson** [comment]
Just went into Developer mode and read the logs while I reproduced. Also cleared the blob_storage directory for the extension before I started in case that was an issue (3.9GB).

Got this error when it reached 100%:
`hostReportError.js:3 Uncaught DOMException: The requested file could not be read, typically due to permission problems that have occurred after a reference to a file was acquired.`

Maybe I wasn't on that tab when it finished?
